BudgetTracker.BudgetSquirrel.WebApi/Auth/ClaimsHelper.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using System.Security.Claims;
+using BudgetTracker.Business.Auth;
+
+namespace BudgetTracker.BudgetSquirrel.WebApi.Auth
```

Comment:
Is this needed? I don't see it referenced anywhere.

Reply:
What are you specifically referencing here?